resource-managers/kubernetes/integration-tests/README.md

```diff
@@ -268,6 +268,22 @@ to the wrapper scripts and using the wrapper scripts will simply set these appro
     </td>
     <td></td>
   </tr>
+  <tr>
+    <td><code>spark.kubernetes.test.driver.cpu</code></td>
```

Comment:
This should be `cores` because you are setting `cores` via `spark.kubernetes.driver.request.cores`.